Chat-Ops-Collector/src/main/java/com/capitalone/dashboard/collector/RestOperationsSupplier.java

```diff
@@ -0,0 +1,18 @@
+package com.capitalone.dashboard.collector;
+
+import com.capitalone.dashboard.util.Supplier;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+import org.springframework.stereotype.Component;
+import org.springframework.web.client.RestOperations;
+import org.springframework.web.client.RestTemplate;
+
+@Component
+public class RestOperationsSupplier implements Supplier<RestOperations> {
+    @Override
+    public RestOperations get() {
+        HttpComponentsClientHttpRequestFactory requestFactory = new HttpComponentsClientHttpRequestFactory();
+        requestFactory.setConnectTimeout(20000);
```

Comment:
will have to refactor all the collectors, it's on a to do list with the admin page story